README.md

```diff
@@ -207,6 +207,8 @@ import { handleActions } from 'redux-actions';
 
 Creates multiple reducers using `handleAction()` and combines them into a single reducer that handles multiple actions. Accepts a map where the keys are passed as the first parameter to `handleAction()` (the action type), and the values are passed as the second parameter (either a reducer or reducer map). The map must not be empty.
 
+If `reducerMap` has a recursive structure, its leaves are used as reducers, and the action type for each leaf is the path to that leaf. If a node's only children are `next()` and `throw()`, the node will be treated as a reducer.
+
```

Comment:
> array form `[payload, meta]`

Isn't this exclusively for action creators? 🤔 
I think `undefined`, `null` & 'fn' as leaves are all possible, as they're just passed to `handleAction`.

I'll add tests for this!